src/testModel.ts

```diff
@@ -89,6 +92,22 @@ export class TestModel {
     this._didUpdate = new vscode.EventEmitter();
     this.onUpdated = this._didUpdate.event;
     this.tag = new this._vscode.TestTag(this.config.configFile);
+    this._spawnTraceViewer = new SpawnTraceViewer(vscode, options.envProvider, this.config);
+
+    this._disposables.push(
+        this._spawnTraceViewer,
+        options.settingsModel.showTrace.onChange(value => {
+          if (!value)
+            this._spawnTraceViewer.close();
+        }),
+    );
+  }
+
+  enabledTraceViewer() {
```

Comment:
I'd call it traceViewer() at this point.

Reply:
Addressed in https://github.com/microsoft/playwright-vscode/pull/509/commits/643d3c37a4d9c71e7d12ecb2e56016e98feccfe9